rhessys/tec/input_new_patch_mult.c

```diff
@@ -42,7 +42,7 @@
 #include "rhessys.h"
 #include "phys_constants.h"
 #define ONE 1.0
-
+#include "params.h"
 
  void  input_new_patch_mult(
```

Comment:
Brian, I have hard time understanding the logic behind it too. Maybe it will be useful for uncertainty analysis when we want to increase or decrease the variable value by 10%, 20%. I haven't used the input_***_mult to calculate the sensitivity. 
Brian, Naomi, I need your suggestions about how to change the code.

Xiaoli